docs/documentation/pages/administrator/020-prerequisites.md

```diff
@@ -39,6 +39,14 @@ for avoiding confusing situations as outlined
 In the end, you won't need to create NGINX ingresses yourself: the Astarte Operator itself will take
 care of this task.
 
+## RabbitMQ (deprecated)
```

Comment:
I'd suggest to do as we do with Scylla (below): list it as an external prerequisite, then reference the deprecated possibility of having it managed by the operator